### ISSUE TITLE
Error in _ftelli64() under Linux

### ISSUE THREAD

**Issue by Worldexe** [Microsoft/CNTK#1722]
I believe there is an error in `CNTK/Source/Common/Include/Platform.h`, in function:
```
inline int _ftelli64(FILE *file)
{
    return ftello(file);
}
```

There must be `int64_t` as return type.
Otherwise, this leads to error if you try to read files >2G.

**Comment by mahilleb-msft** [comment]
By chance, this was addressed by e8a8de17deba6238497d0501c3282db6a8907c28 yesterday. I guess we can close this here?

**Comment by Worldexe** [comment]
Yeah, thx.
